mock-core-configs/mock-core-configs.spec

```diff
@@ -24,7 +24,7 @@ Provides: mock-configs
 # distribution-gpg-keys contains GPG keys used by mock configs
 Requires:   distribution-gpg-keys >= 1.110
 # specify minimal compatible version of mock
-Requires:   mock >= 5.4.post1
+Requires:   mock >= 6.2
```

Comment:
This will not work with testing releases, currently we have `6.1.test` (try `tito build --test`), hence we should have `>= 6.1.test`.